### ISSUE TITLE
 jakarta.servlet.ServletContainerInitializer: Provider org.omnifaces.jwt.servlet.RolesDeclarationInitializer not found

### ISSUE THREAD

**Issue by hantsy** [eclipse-ee4j/glassfish#25043]
There are several similar exceptions in my Glassfish v8 server logs. I did not use omnifaces here.

## Environment Details
* GlassFish Version (and build number): 8.0.0-M6
* JDK version: 21
* OS: Windows 10 Pro
* Database: Built-in Derby

----------

## Problem Description

```
java.util.ServiceConfigurationError: jakarta.servlet.ServletContainerInitializer: Provider org.omnifaces.jwt.servlet.RolesDeclarationInitializer not found
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:593)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1219)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393)
	at org.apache.catalina.core.StandardContext.loadServletContainerInitializers(StandardContext.java:4624)
	at org.apache.catalina.core.StandardContext.callServletContainerInitializers(StandardContext.java:4538)
```

**Comment by dmatej** [comment]
@arjantijms Do you know why is it happening and how get rid of that? I have seen it too when running TCKs.

**Comment by arjantijms** [comment]
It is already fixed in the 8.0 branch.

It was because the JWT implementation referenced some old dependency that could not be found in 8.0.

**Comment by arjantijms** [comment]
Fixed by: https://github.com/eclipse-ee4j/glassfish/pull/25035
